### PR TITLE
Vfep 903 - add guardian certification/signature to 1995 spool file

### PR DESCRIPTION
**Pull request by GcioGregg**
Vfep 903 - add guardian certification/signature to 1995 spool file

## Summary

- *This work is behind a feature toggle (flipper): YES*
- added conditional statement by signature if Guardian is present
- VFEP
- feature flag will be removed upon user acceptance

## Related issue(s)

- [vfep-903](https://jira.devops.va.gov/browse/VFEP-903)
- [github 16264](https://github.com/department-of-veterans-affairs/vets-api/issues/16264)

## Testing done

- [x] *New code is covered by unit tests*
- 1995 spool file did not contain new statement when guardian present
- generate 1995 spool file and confirm new statement is there when guardian is present
- user testing of all recent 1995 spool changes on staging, once approved feature flag will be removed

## What areas of the site does it impact?
impacts 1995 spool file

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by nfstern02 — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -1283,3 +1283,4 @@ RUBY VERSION
 
 BUNDLED WITH
    2.4.9
+
```
Comment (ericboehs) [inline Gemfile.lock:1286]: Can we remove this extra new line?

File: app/sidekiq/education_form/templates/1995.erb [hunk 2]
```diff
@@ -184,10 +184,14 @@ Signature/Title/Branch of Armed Forces Education Service Officer     Date
...
+      You are the parent, guardian, or custodian of the applicant
 <% else -%>
       Certification and Signature of Applicant
+<% end -%>
 Signature of Applicant                                               Date
 <% end -%>
 <% end -%>
 
-<%= parse_with_template_path('footer') %>
+<%= parse_with_template_path('footer') %>
```
Comment (ericboehs) [inline app/sidekiq/education_form/templates/1995.erb:197]: This file is missing a final new line. If you're using VS Code: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

**Review by ericboehs — CHANGES_REQUESTED** [review]
newline stuff

**Review by ryan-mcneil — APPROVED** [review]
👍

**Review by nfstern02 — DISMISSED** [review]

**Review by bradbergeron-us — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
LGTM

**Pull request merged**